package.json

```diff
@@ -95,8 +95,10 @@
   },
   "dependencies": {
     "@resin.io/valid-email": "^0.1.0",
+    "@types/bluebird": "3.5.25",
     "@types/dockerode": "2.5.5",
     "@types/mixpanel": "2.14.0",
+    "@types/request": "2.48.1",
```

Comment:
These `@types` additions are merely incidental to this PR. My VS Code was nagging me with red underlines that the typescript code was missing out on type definitions and I agreed.